src/cpp/rtps/builtin/discovery/endpoint/EDPSimple.cpp

```diff
@@ -385,10 +367,6 @@ bool EDPSimple::create_sedp_secure_endpoints()
         ratt.endpoint.multicastLocatorList = this->mp_PDP->getLocalParticipantProxyData()->m_metatrafficMulticastLocatorList;
         //ratt.endpoint.remoteLocatorList = m_discovery.initialPeersList;
         ratt.endpoint.durabilityKind = TRANSIENT_LOCAL;
-        ratt.times.heartbeatResponseDelay.seconds = 0;
-        ratt.times.heartbeatResponseDelay.fraction = 0;
-        ratt.times.initialAcknackDelay.seconds = 0;
-        ratt.times.initialAcknackDelay.fraction = 0;
         ratt.endpoint.security_attributes().is_submessage_protected = part_attr.is_discovery_protected;
```

Comment:
Maybe use specific values for all the ratt.times fields? Even better, have a const for it at the top of the file?

Reply:
Done